### ISSUE TITLE
Micro rust plugin page seems to be missing

### ISSUE THREAD

**Issue by shoccho** [contradictioned/areweideyet#104]
The link for the "[micro-rust-plugin](https://github.com/tommyshem/micro-rust-plugin)" for micro text editor seems to be broken.

**Comment by contradictioned** [comment]
Yes, seems the author does not use micro anymore: https://github.com/tommyshem/micro-editor-stuff
